lib/rubocop/cop/util.rb

```diff
@@ -148,6 +148,10 @@ def move_pos(src, pos, step, condition, regexp)
         pos
       end
 
+      def looking_at(range, regexp)
+        range.source_buffer.source.match(/\G#{regexp}/, range.begin_pos)
```

Comment:
Wrong.  Compare `/\Gz/.match("xyz", 1)` with `/z/.match("xyz", 1)`.  I expected the name `looking_at` was self-explanatory.